lib/standard/numeric.nit

```diff
@@ -30,9 +30,11 @@ redef class Text
 	# assert "0.0".to_n == 0.0
 	# assert ".12345".to_n == 0.12345
 	# assert "12345".to_n == 12345
+	# assert "".to_n == 0
 	# ~~~~
 	fun to_n: Numeric
 	do
+		if is_empty then return 0
```

Comment:
It was what I expected too but I get an assert failed with:

~~~
print "".to_i
~~~

The error details:
~~~
/home/xymus/projects/nit/lib/standard/text/flat.nit:53,3--19: Runtime error: Assert failed
			assert index < ln
			^
,---- Stack trace -- - -  -
| flat#FlatText#char_to_byte_index (/home/xymus/projects/nit/lib/standard/text/flat.nit:53,3--19)
| flat#FlatText#[] (/home/xymus/projects/nit/lib/standard/text/flat.nit:87,46--70)
| fixed_ints#Text#is_int (/home/xymus/projects/nit/lib/standard/fixed_ints.nit:889,9--14)
| fixed_ints#Text#to_i (/home/xymus/projects/nit/lib/standard/fixed_ints.nit:904,10--20)
| f#Sys#main (/tmp/f.nit:1,7--13)
| kernel#Sys#run (/home/xymus/projects/nit/lib/standard/kernel.nit:296,13--16)
`------------------- - -  -
~~~

Should I move my test to `to_i`?